modules/examples/todolist-service/src/main/scala/todo/runtime/implicits.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2017-2018 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package todo
+package runtime
+
+import java.util.Properties
+
+import cats.effect._
+import com.zaxxer.hikari.{HikariConfig, HikariDataSource}
+import doobie._
+import doobie.hikari._
+
+object implicits extends ProductionImplicits
+
+trait ProductionImplicits {
+
+  implicit def buildHikariTransactor[F[_]: Async](): Transactor[F] =
```

Comment:
We can require the doobie transactor in our Persistence handlers, but perhaps we don't need any particular implementation. Clients of your library can specify its own Transactors if they are using the PersistenceHandlers that you are providing as part of your library.

Reply:
Totally agree! It just slip in the PR :grin: